controllers/apply/form-router-next/index.js

```diff
@@ -172,17 +172,23 @@ function initFormRouter({
     /**
      * Route: Eligibility
      */
-    router.use(
-        '/eligibility',
-        require('./eligibility')(eligibilityBuilder, formId)
-    );
+    if (eligibilityBuilder) {
+        router.use(
+            '/eligibility',
+            require('./eligibility')(eligibilityBuilder, formId)
+        );
+    }
 
     /**
      * Route: Start application
      * Redirect to eligibility checker
      */
     router.get('/start', function(req, res) {
-        res.redirect(`${req.baseUrl}/eligibility/1`);
+        if (eligibilityBuilder) {
```

Comment:
The new form doesn't have an eligibility checker so making this optional and start a new application if we don't have a checker step.